back/boxtribute_server/business_logic/tag/crud.py

```diff
@@ -80,19 +80,23 @@ def update_tag(
 
     with db.database.atomic():
         if object_type_for_deletion is not None:
-            TagsRelation.delete().where(
-                (TagsRelation.object_type == object_type_for_deletion)
-                & (TagsRelation.tag == id)
+            TagsRelation.update(deleted_on=utcnow(), deleted_by=user_id).where(
```

Comment:
pls use timestamp from decorator. 
Pls change also for boxes.
Pls check if timestamp in history is the same during creation as the created_on field